sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -526,6 +526,56 @@ public void invalidDataEncryptionKeyAlgorithm() {
         }
     }
 
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void patchItem() {
```

Comment:
Please add patch state cache test in test "crudQueryStaleCache", check  how we doing other operation including batch

Reply:
Added the test